app/helpers/application_helper/toolbar/firmware_registry_center.rb

```diff
@@ -5,7 +5,7 @@ class ApplicationHelper::Toolbar::FirmwareRegistryCenter < ApplicationHelper::To
       button(
         :firmware_registry_reload,
         'fa fa-refresh fa-lg',
-        N_('Refresh this page'),
+        nil,
```

Comment:
removing the title instead of the text here?

Reply:
Well, if we want to keep the refresh icon but not the text, then sure. Is this the case?

Reply:
keeping the icon, not the text is fine, that's what happens with the other refresh buttons

but this one removes the title, keeps the text and the icon :)

Reply:
@himdel Fixed